client-node/index.js

```diff
@@ -8,7 +8,10 @@ const DEFAULT_OPTIONS = {
 
 export class ClientNode extends BaseNode {
   constructor(nodeId, log, connection, options = {}) {
-    options = { ...DEFAULT_OPTIONS, ...options }
+    options = { ...options }
+    options.fixTime ??= DEFAULT_OPTIONS.fixTime
```

Comment:
Can you avoid `??=` to be able to release it without major changes (we need to support Node.js)